source/ranges.tex

```diff
@@ -194,6 +194,14 @@
   template<class T>
     inline constexpr bool enable_borrowed_range<ref_view<T>> = true;
 
+  // \ref{range.owning.view}, owning view
```

Comment:
"owning view" -> "\tcode{owning_view}" or "class \tcode{owning_view}" ("owning view" has no meaning). Same for other headers below.

Reply:
Please see the headers for "take view", "drop view", and others. Not doing this.